contracts/governance_admin/tests/create-pool.rs

```diff
@@ -175,6 +175,9 @@ impl<'a> CreatePoolTestSuite<'a> {
 
         println!("Running success case: Refund amount post rejected proposal");
         self.test_refund_for_rejected_proposal();
+
+        println!("Running success case: Refund amount post vetoed proposal");
+        self.test_refund_for_vetoed_proposal();
```

Comment:
Add new test case where only few validators vote and majority of them are `veto`.
Still the proposal deposit should be refunded, as the proposal is rejected due to not reaching the quorum.

Reply:
Decided not to handle the case since that information is not accessible in the proposal state.